vita3k/renderer/src/texture_format.cpp

```diff
@@ -180,10 +180,10 @@ void tiled_texture_to_linear_texture(uint8_t *dest, const uint8_t *src, uint16_t
     for (uint16_t y = 0; y < height; y++) {
         for (uint16_t x = 0; x < width; x++) {
             // Calculate texel address in title
-            const uint16_t texel_offset_in_title = (x & 0b11111) | ((y & 0b11111) << 4);
+            const uint16_t texel_offset_in_title = (x & 0b11111) | ((y & 0b11111) << 5);
             const uint16_t tile_address = (x & 0b111111100000) + width_in_tiles * (y & 0b111111100000);
 
-            uint32_t offset = ((tile_address << 9) | (texel_offset_in_title)) * (bits_per_pixel >> 3);
+            uint32_t offset = ((tile_address << 5) | (texel_offset_in_title)) * (bits_per_pixel >> 3);
```

Comment:
this value is for fix crash, using 9/8 crash, using 7/6 get broken render with screen splitted